e2e/common/common.go

```diff
@@ -16,9 +17,19 @@ import (
 	"github.com/loomnetwork/loomchain/e2e/node"
 )
 
+const (
+	loomExeEv = "LOOMEXE_PATH1"
+	loomExe2Ev = "LOOMEXE_ALTPATH"
+	minValidatorsEv = "MIN_VALIDATORS"
+	minAltValidatorsEv = "MIN_ALT_VALIDATORS"
```

Comment:
Not entirely sure what these min validator vars are for.

Reply:
Currently, the number of validators are set in the `??_test.go` file and passed into common.NewConfig or otherwise hardcoded. Typically 4 and 0 for the two builds. 
These two environmental variables allow the number of nodes in using each of the two build to be set from the calling script rather than hardcoded.